api/resources_portal/migrations/0005_auto_20200501_2022.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 2.2.12 on 2020-05-01 20:22
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("resources_portal", "0004_auto_20200501_1518"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="ShippingRequirements",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True, primary_key=True, serialize=False, verbose_name="ID"
+                    ),
+                ),
+                ("created_at", models.DateTimeField(auto_now_add=True)),
+                ("updated_at", models.DateTimeField(auto_now=True)),
+                ("needs_shipping_address", models.BooleanField(default=False)),
+                ("sharer_pays", models.BooleanField(default=False)),
+                ("accepts_ups_code", models.BooleanField(default=False)),
```

Comment:
hey so `accepts_ups_code` and `accepts_fedex_code` should be `accepts_shipping_code` and then we will tell them to clarify in the restrictions.